web-console/osio-planner/manage-label.md

```diff
@@ -0,0 +1,8 @@
+# Manage Label
+## Sc-1: Planner Quick preview ##
+![Planner quick preview](img/planner-view1)
```

Comment:
@muruGanesan this is great.  Thanks for doing multiple commits.  I would suggest trying to rename your image to have a png suffix.  That way you will have to rename a file and update the markdown to access it appropriately.